### ISSUE TITLE
cam port "reversed" 

### ISSUE THREAD

**Issue by Gabbajoe** [bigtreetech/Manta-M5P#18]
Hi,
by looking at the manual and testing and scratching my head it turns out the cam port on the Manta M5P is revers compared to a regular Raspberry Pi. This means you can't use the regular Type B ribbon cables that the cams come with. If you use the Type B cable your CM4 module will just crash(short between 3,3V and ground) and the activity LED will flicker very dim and nothing is happening. It would be nice to have this hint reflect in the manual in written form and not just a picture on Page 18. Took me 2 hours and reading this issue https://github.com/bigtreetech/Manta-E3EZ/issues/5 from the Manta E3EZ to try to use a Type A cable instead of a Type B cable.
To verify the Type A cable theory I  did the suggested from here https://github.com/bigtreetech/Manta-E3EZ/issues/5#issuecomment-1479877946 After that the Raspberry PI Cam V3 is working fine. 

Further more the CSI1 Camera section on page 25/35 is a red herring in my opinion. 
Original text:

CSI1 Camera
The DSI1 driver downloaded in DSI1 Display also includes the
CSI1 driver. If you just want to install the CSI1 driver, not DSI1, please find the
driver you want to use at https://datasheets.raspberrypi.com/licence.html and
download it in the boot folder of CM4 and rename it to dt-blob.bin, then refer
to the settings here: https://projects.raspberrypi.org/en/projects/getting-started-
with-picamera/

The last link will help you not at all!

This link will help you: https://www.raspberrypi.com/documentation/computers/compute-module.html#quickstart-guide-display-and-cameras

Why not simply provide the command?
sudo wget https://datasheets.raspberrypi.com/cmio/dt-blob-disp1-only.bin -O /boot/firmware/dt-blob.bin

Also interesting is that they write on the Official Website Compute Module website to put the the binary firmware into the path /boot/firmware/ and not just /boot/. But /boot/ works too after my testing. So no issue here.

Cheers,
Gabbajoe.

**Comment by WJN516** [comment]
The wget link appear to no longer function and gives a 404 error when using.
I was able to download the driver from the wayback machine here: https://web.archive.org/web/20230521063914/https://datasheets.raspberrypi.com/cmio/dt-blob-cam1.bin

I'm very unfamiliar with wget so I just used scp to copy it to the home folder of cm4, then copied it to /boot/firmware, then renamed it to dt-blob.bin, its possible that you could just use the wayback link in place of the original, I don't know.

With this and the type A FFC cable, I was finally able to get my camera to function.

**Comment by chris-red** [comment]
Could this explain why when I plugged in a PI camera with the supplied cable I smelt burning and the camera had let the magic smoke out? I'm reluctant to buy another and try again...

**Comment by WJN516** [comment]
Yes, when I plugged the camera in with the cable that it came with, it smelled of magic smoke and the auto-focus was fried, and my camera is now perpetually out of focus. I only had it powered on for a few seconds, so that may be why my camera still barely functions and yours does not.
